universal-login-relayer/lib/routes/wallet.ts

```diff
@@ -27,10 +27,26 @@ messageHandler : MessageHandler) => async (req : Request, res : Response, next :
   }
 };
 
-export const getStatus = (transactionContractService: MessageHandler) => async (req: Request, res: Response, next: NextFunction) => {
+export const getStatus = (messageHandler: MessageHandler) => async (req: Request, res: Response, next: NextFunction) => {
   try {
     const {hash} = req.params;
-    const status = await transactionContractService.getStatus(hash);
+    const status = await messageHandler.getStatus(hash);
+    res.status(200)
+      .type('json')
+      .send(JSON.stringify(status));
+  } catch (err) {
+    next(err);
+  }
```

Comment:
#nextPR 
to my best knowlage  `try { ...} catch (err) {next(err);}` is only needed for asynchronious code, so we can skip it there

https://expressjs.com/en/guide/error-handling.html

Reply:
Ticket created